ide/src/reducer.ts

```diff
@@ -881,6 +882,46 @@ async function runProgramAsync(state: State) : Promise<void> {
   }
 }
 
+function copyFileSync(src: string, dst: string) {
+  // (browser)fs doesn't have copyFileSync(). Simulate using {read,write}FileSync()
+  const data = fs.readFileSync(src);
+  fs.writeFileSync(dst, data);
+}
+
+async function runExamplarAsync(state: State) : Promise<void> {
+  const {
+    typeCheck, runKind, currentFile, currentFileContents,
+  } = state;
+  // eslint-disable-next-line
```

Comment:
Side note as I'm reading this, best if we only use `eslint-disable` for lines that use `await` in loops. Otherwise best to fix the lint error.